code/modules/admin/verbs/randomverbs.dm

```diff
@@ -64,7 +64,7 @@
 		to_chat(usr, "The person you are trying to contact is not wearing a headset.")
 		return
 
-	if (!sender)
+	if(!sender)
```

Comment:
Spaces between if statements should not be changed for no reason. See code standards for more details.